simplified-tests/src/test/java/org/nypl/simplified/tests/books/controller/ProfilesControllerContract.kt

```diff
@@ -458,6 +458,14 @@ abstract class ProfilesControllerContract {
               get() = "Author"
             override val sortByTitle: String
               get() = "Title"
+            override val show: String
+              get() = "Show"
+            override val showAll: String
+              get() = "All"
+            override val showEBooks: String
+              get() = "eBooks"
+            override val showAudioBooks: String
+              get() = "AudioBooks"
```

Comment:
Do `showEBooks` and `showAudioBooks` get used? It's not clear to me why they were added.

Reply:
No, they are not used. I forgot to remove this as well